addons/pf-maint.pl

```diff
@@ -120,36 +120,66 @@ =head1 SYNOPSIS
 
 our $BINARIES_SIGN_KEY_ID = "A0030E2C";
 
+our $step = 0;
+
 my $base = $BASE_COMMIT || get_base();
 
 die "Cannot base commit\n" unless $base;
 
+$step++;
+print "=" x 110 . "\n";
+print "Step $step: Patching of text based codefiles\n";
+
 print "Currently at $base\n";
 
 my $head = $COMMIT || get_head();
 if ($base eq $head) {
-    print "Already up to date\n";
-    exit 0;
+    print "Already up to date for text based patches\n";
+} 
+else {
+    print "Text based patche(s) available\n";
+    exit 1 if defined($test);
```

Comment:
exit 0 instead of 1 since it not a failure

Reply:
It has to be "exit 1" since it has to be a failure for the monit check to send us an alert when a patch is available.

Reply:
Ok